src/components/Congress/Motions/index.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Card from '../../Card';
+import styles from './styles.module.scss';
+import Button from '../../Button/Button';
+
+// REDUX
+import { congressActions } from '../../../redux/actions';
+import {
+  congressSelectors,
+  blockchainSelectors,
+} from '../../../redux/selectors';
+
+export default function Motions() {
+  const dispatch = useDispatch();
+  const motions = useSelector(congressSelectors.congressMotions);
+  const sender = useSelector(blockchainSelectors.userWalletAddressSelector);
+
+  useEffect(() => {
+    dispatch(congressActions.getMotions.call());
+  }, [dispatch]);
+
+  return (
+    <div>
+      {/* eslint-disable-next-line react/prop-types */}
+      {motions.map(({ proposal, proposalOf, voting }) => {
```

Comment:
Extract it to a separate component (can be in the same file), so that we can have a more readable:
```
{
  motions.map(({ proposal, proposalOf, voting }) =>
    <Motion key={FIXME} proposal={proposal} proposalOf={proposalOf} voting={voting} />
  )
}
```